### PR TITLE
Zh cn/update installation

### PR DESCRIPTION
**Pull request by Zane5**
Translate en/community/weblogs/ (zh_cn)
screen shot 2015-06-05 at 19 23 55

Related [#1074](https://github.com/ruby/www.ruby-lang.org/issues/1074)

### CODE REVIEW

**Comment by JuanitoFatas** [comment]
@pmq20 @hlcfan @liuzxc @tzwm @happybai @diguage @alex-marmot @itsliupeng
@cassiuschen @1ternal @killernova @VincentZhao @reyesyang @jun1st @springwq
@Huibean @junzh0u @zigzagpig @42thcoder @GaaraLi @shouxian @johnnyhappy365
@aidewoode @Yunfei1982 @Airmacho @cursorzz @isjia @debbbbie @AndyHelix
@Zane5 @VirgilMing @boyishwei @dandananddada @jundechen @mohongshu

Please review.

File: zh_cn/community/weblogs/index.md [hunk 1]
```diff
@@ -4,4 +4,56 @@ title: "博客 - Ruby 官方网站"
...
+* [**Matz' Blog**][11] Ruby 的创始人写的日文博客。
+  即使你一点儿都看不懂，知道他就很好。
+
+### 请传播这个词
+
+如果你已经有一个关于 Ruby 的博客，在 [del.icio.us][12] 
+上面打上 *ruby* 的标记是很明智的。
+你也可以联系上面提到的博客，如果他们对你的博客内容感兴趣。
+（显然，如果和 Rails 不相干，*Riding Rails* 
+团队会不感兴趣，或许你不知道这一点。）
```
Comment (hlcfan) [inline zh_cn/community/weblogs/index.md]: Is there a reason to start a new line here?

File: zh_cn/community/weblogs/index.md [hunk 2]
```diff
@@ -4,4 +4,56 @@ title: "博客 - Ruby 官方网站"
 lang: zh_cn
 ---
 
-{% include out-of-date.html %}
+Ruby 相关的博客经过去一年的爆发和相关职位充分发展，
+你能发掘出数以百计的博客，他们共享 Ruby 代码，介绍新技术，
+或者预测 Ruby 的未来。
```
Comment (hlcfan) [inline zh_cn/community/weblogs/index.md]: Is there a reason to start a new line?

**Comment by hlcfan** [comment]
All good except some minor problems.

File: zh_cn/community/weblogs/index.md (hunk 2 above)
Comment (Zane5) [inline zh_cn/community/weblogs/index.md]: qq 20150611154536

I refer to the English original, but it is not necessary to add some new lines in chinese.

**Comment by JuanitoFatas** [comment]
Superseded by #1082.

Thank you for your contribution @Zane5, if you have further improvements please start a new branch from current master and open a new pull request. Thanks!

**Comment by stomar** [comment]
You mean #1127.

**Comment by JuanitoFatas** [comment]
> You mean #1127.

Correct. :sweat:

**Pull request closed without merging**